Classes/rooms/rooms.py

```diff
@@ -0,0 +1,63 @@
+#rooms
+import random
+
+class Room(object):
+    all_rooms=0
+    rooms=[]
+    def __init__(self,name,what_type):
+        self.all_rooms+=1
+        self.what_type=what_type
+
+    def __init__(self):
+        pass
+
+    def create_room(self,name,what_type):
+        self.what_type=what_type
```

Comment:
Make the variable names more descriptive.
Avoid using variable names such as 'what_type', 'as_many'